frontend/public/components/monitoring/receiver-forms/email-receiver-form.tsx

```diff
@@ -136,7 +141,7 @@ export const Form: React.FC<FormProps> = ({ globals, formValues, dispatchFormCha
                   }
                 />
                 <div className="help-block" id="email-hello-help">
-                  The hostname to identify to the SMTP server.
+                  {t('email-receiver-form~The hostname to identify to the SMTP server.')}
```

Comment:
In docs, we use host name per the IBM style guide. Another likely out of scope comment, but something we might consider syncing up on later :smile: